training-yolov8-classification/picsellia/utils.py

```diff
@@ -0,0 +1,320 @@
+from typing import Tuple, Any
+
+import numpy
+from PIL import Image
+import numpy as np
+import os
+from picsellia.sdk.experiment import Experiment
+from picsellia.sdk.dataset_version import DatasetVersion
+
+from pycocotools.coco import COCO
+import shutil
+from picsellia.types.enums import AnnotationFileType
+from picsellia.exceptions import ResourceNotFoundError
+from picsellia.sdk.asset import MultiAsset
+from picsellia.sdk.label import Label
+from ultralytics.yolo.engine.model import YOLO
+
+
+def get_train_test_eval_datasets_from_experiment(
+    experiment: Experiment,
+) -> tuple[bool, bool, DatasetVersion, DatasetVersion, DatasetVersion]:
+    number_of_attached_datasets = len(experiment.list_attached_dataset_versions())
+    has_three_datasets, has_two_datasets = False, False
+    if number_of_attached_datasets == 3:
+        has_three_datasets = True
+        train_set, test_set, eval_set = _get_three_attached_datasets(experiment)
+    elif number_of_attached_datasets == 2:
+        has_two_datasets = True
+        train_set, test_set, eval_set = _transform_two_attached_datasets_to_three(
+            experiment
+        )
+    elif number_of_attached_datasets == 1:
+        print(
```

Comment:
Change all the prints with logging.info(...) so we can implement a logger afterward